doc/source/ecosystem.rst

```diff
@@ -327,6 +327,26 @@ PyTables, h5py, and pymongo to move data between non pandas formats. Its graph
 based approach is also extensible by end users for custom formats that may be
 too specific for the core of odo.
 
+`Pandaral·lel <https://github.com/nalepae/pandarallel>`__
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pandaral·lel provides a simple way to parallelize your pandas operations on all your CPUs by changing only one line of code.
```

Comment:
Pandaral·lel typo?

Reply:
Actually not, the point is inspired from [Avinguda del Paral·lel](https://en.wikipedia.org/wiki/Avinguda_del_Paral%C2%B7lel) in Barcelona, Spain.

This point is also present on the [GitHub page of the project](https://github.com/nalepae/pandarallel).

I removed it anyway.